CS483/CS483/Kartaclysm/StateMachine/Gameplay/StateRacing.cpp

```diff
@@ -133,11 +133,7 @@ HeatStroke::GameObject* Kartaclysm::StateRacing::GenerateRacer
 	std::string strRacerDefinitionFile = "CS483/CS483/Kartaclysm/Data/Racer/racer.xml";
 	HeatStroke::GameObject* pRacer = m_pGameObjectManager->CreateGameObject(strRacerDefinitionFile, p_strGuid);
 
-	if (p_strGuid == "Player0")
-	{
-		HeatStroke::GameObject* pHUD = m_pGameObjectManager->CreateGameObject("CS483/CS483/Kartaclysm/Data/Racer/hud.xml", p_strGuid + "_HUD");
-	}
-	// TODO: make the HUD work for multiple players.
+	CreateHUDForRacer(p_strGuid);
```

Comment:
Is there any benefit to generate HUDSs once at a time? Could they not all be made at once in CreateHudForRacers()?